src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -702,7 +712,7 @@ export default class CDPipeline extends Component<CDPipelineProps, CDPipelineSta
     }
 
     renderHeader() {
-        const title = this.isWebhookCD
+        const title = this.isWebhookCD && this.props.match.params.workflowId === '0'
```

Comment:
What is this var used for? It's a boolean so can we rename it according to usage?